client/quota.go

```diff
@@ -74,21 +74,37 @@ type QuotaValues struct {
 	Journal *QuotaJournalValues `json:"journal,omitempty"`
 }
 
-// EnsureQuota creates a quota group or updates an existing group.
-// The list of snaps can be empty.
-func (client *Client) EnsureQuota(groupName string, parent string, snaps, services []string, constraints *QuotaValues) (changeID string, err error) {
+type EnsureQuotaOptions struct {
+	// Parent is used to assign a Parent quota group
+	Parent string
+	// Snaps is the Snaps that should be added to the quota group
+	Snaps []string
+	// Services is the snap Services that should be added to the quota group
```

Comment:
```suggestion
	// Services that should be added to the quota group
```

The simplified version doesn't mention "snap services". IMO the "snap" part is implicit in this context but it's subjective so it's up to you